src/environments/environment.ts

```diff
@@ -1,3 +1,15 @@
+import { Profile } from '../app/auth/profiles/profiles.component';
+
 export const AppConfig = {
-  production: false
+  production: false,
+  clientId: '0cbc5e651d8b01e36687',
```

Comment:
Should we store these variables somewhere else, maybe in the form of `SECRETS` 

Else, it might be possible for others to get access to our client ids and possibly misuse them!

Reply:
@seanlowjk, any idea where else to store it? We need the `clientId` on the client side for the proxy server to identify the client.

Reply:
Not sure whether it is possible in Electron, because everything is done in the client side. And we can't pre-define the environment variables on their machine.
But for the web-app, I think we can make use of github secrets to hide the clientId.

Reply:
I actually found out that clientID may not need to be a secret after all according to https://stackoverflow.com/a/37945932
I made the misconception about clientID and clientSecret ><